docs/recipes/managing_multiple_clusters.adoc

```diff
@@ -0,0 +1,164 @@
+== Managing multiple clusters
+
+CBShell is a powerful tool that can be used to interact with fleets comprised of a mix of self-managed and Capella clusters.
+Say we have the following four clusters registered with CBShell:
+
+```
+👤 Charlie 🏠 obligingfaronmoller in ☁️ default._default._default
+> cb-env managed
+╭───┬────────┬───────┬────────────────────────┬───────────────┬──────────────────────┬─────────────────╮
+│ # │ active │  tls  │       identifier       │   username    │ capella_organization │     project     │
+├───┼────────┼───────┼────────────────────────┼───────────────┼──────────────────────┼─────────────────┤
+│ 0 │ false  │ true  │ meekjoycekreynolds     │ Administrator │ my-org               │ CBShell Testing │
+│ 1 │ false  │ false │ local                  │ Administrator │                      │                 │
+│ 2 │ false  │ true  │ bronzegeraldjaysussman │ Administrator │ my-org               │ CBShell Testing │
+│ 3 │ true   │ true  │ obligingfaronmoller    │ Administrator │ my-org               │ CBShell Testing │
+╰───┴────────┴───────┴────────────────────────┴───────────────┴──────────────────────┴─────────────────╯
```

Comment:
Minor improvement if you want, give them identifiers that match a narrative.  like "localdev" and "systemtest" and "prod" and "ci".

Reply:
Done